apps/studio/components/interfaces/APIKeys/ApiKeysIllustrations.tsx

```diff
@@ -120,7 +120,7 @@ export const ApiKeysComingSoonBanner = () => {
         <div className="mt-4">
           <Button type="default" icon={<Github />}>
             <a
-              href="https://github.com/orgs/supabase/discussions/29260"
+              href="https://github.com/simplyblock/vela-studio/discussions/29260"
```

Comment:
I think this doesn't make sense as it'll get to a discussion which doesn't exist. I wonder if it'd be better to remove it altogether. It'll be a 404 otherwise.